rq/worker.py

```diff
@@ -1098,17 +1105,23 @@ def perform_job(self, job: 'Job', queue: 'Queue'):
         """Performs the actual work of a job.  Will/should only be called
         inside the work horse's process.
         """
+        self.log.debug("Starting process to perform a job...")
         push_connection(self.connection)
+        self.log.debug("Connection pushed.")
 
         started_job_registry = queue.started_job_registry
+        self.log.debug("Started Job Registry set.")
 
         try:
+            self.log.debug("Preparing for execution...")
```

Comment:
I think we can remove this statement, there's already a logging statement in `prepare_job_execution`